.bazelci/presubmit.yml

```diff
@@ -0,0 +1,24 @@
+---
+bazel: 2.1.0
+platforms:
+  ubuntu1804:
+    run_targets:
+    - "..."
+    build_targets:
+    - "//src/com/google/daggerquery:dagger_query_plugin"
+    test_targets:
+    - "//tests/com/google/daggerquery/plugin:GraphConverterTest"
```

Comment:
You might want to consider using a wildcard suffix, e.g. `"//tests/..."` so that you don't have to maintain this file every time a new target is added to the codebase.

Reply:
Done, thanks!